tests/foreman/ui/test_host.py

```diff
@@ -2985,3 +2987,147 @@ def test_positive_manage_repository_sets(
+    domain = target_sat.api.Domain(
+        organization=[module_org.id], location=[module_location]
+    ).create()
+    subnet = target_sat.api.Subnet(
+        organization=[module_org.id], location=[module_location], domain=[domain]
+    ).create()
+    architecture = target_sat.api.Architecture().create()
+    ptable = target_sat.api.PartitionTable(
+        organization=[module_org.id], location=[module_location]
+    ).create()
+    operatingsystem = target_sat.api.OperatingSystem(
+        architecture=[architecture], ptable=[ptable]
+    ).create()
+    medium = target_sat.api.Media(
+        organization=[module_org.id], location=[module_location], operatingsystem=[operatingsystem]
+    ).create()
+    lce = (
+        target_sat.api.LifecycleEnvironment(name="Library", organization=module_org.id)
+        .search()[0]
+        .read()
+        .id
+    )
+    cv = target_sat.api.ContentView(organization=module_org).create()
+    cv.publish()
+
+    # create hostgroup
+    hostgroup_name = gen_string('alpha')
+    target_sat.api.HostGroup(
+        name=hostgroup_name,
+        architecture=architecture,
+        domain=domain,
+        subnet=subnet,
+        location=[module_location.id],
+        medium=medium,
+        operatingsystem=operatingsystem,
+        organization=[module_org],
+        ptable=ptable,
+        lifecycle_environment=lce,
+        content_view=cv,
+        content_source=1,
+    ).create()
+
+    with target_sat.ui_session() as session:
+        session.organization.select(org_name=module_org.name)
+        session.location.select(loc_name=module_location.name)
+        session.computeresource.create(
+            {
+                'name': cr_name,
+                'provider': FOREMAN_PROVIDERS['vmware'],
+                'provider_content.vcenter': vmware.hostname,
+                'provider_content.user': settings.vmware.username,
+                'provider_content.password': settings.vmware.password,
+                'provider_content.datacenter.value': settings.vmware.datacenter,
+                'locations.resources.assigned': [module_location.name],
+                'organizations.resources.assigned': [module_org.name],
+            }
+        )
+        session.hostgroup.update(hostgroup_name, {'host_group.deploy': cr_name + " (VMware)"})
+
+        cr_vm_names = [settings.vmware.vm_name, 'phoenix-testing-guest-rhel-8']
+        vm_names_with_domains = [f'{name.replace(".", "")}.{domain.name}' for name in cr_vm_names]
+
+        # Import VMs from VMware compute resource
+        for cr_vm_name, vm_name_with_domain in zip(
+            cr_vm_names, vm_names_with_domains, strict=False
+        ):
+            session.computeresource.vm_import(
+                cr_name,
+                cr_vm_name,
+                hostgroup_name,
+                module_location.name,
+                name=cr_vm_name.replace('.', ''),
+            )
+            assert session.all_hosts.search(vm_name_with_domain)
```

Comment:
Since this test mirrors the exact steps of test_positive_virt_card except for the disassociation step, would it make sense to incorporate the disassociation check into test_positive_virt_card itself? That way, we can avoid redundancy and keep the test suite concise

Reply:
Good point, but I would still maybe keep this test in test_host (Host-Content) component, as this is meant as a coverage for new Hosts/Host-Content component feature